arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/CreateSection.java

```diff
@@ -215,8 +217,13 @@ public CreateSection(Hub hub) {
         .scheduleForContextReEntry(
             this, hub.currentFrame()); // To get the success bit of the CREATE(2)
 
+    requiresRomLex = true;
```

Comment:
Only the `non-exceptional, non-aborting, non-failing, non-emptyInitCode create` case requires the `ROM_LEX`